analyses/molecular-subtyping-pathology/pathology_free_text-subtyping-meningioma.Rmd

```diff
@@ -26,7 +26,10 @@ library(magrittr)
 
 ```{r}
 data_dir <- file.path("..", "..", "data")
-histologies_file <- file.path(data_dir, "pbta-histologies.tsv")
+results_dir <- file.path("results")
+histologies_file <- file.path(data_dir, "pbta-histologies-base.tsv")
+compiled_mol_subtypes_pathology_clinical_file <- file.path(results_dir,
```

Comment:
don't love the long name `compiled_mol_subtypes_pathology_clinical_file`, but not a huge deal - let's get this in